packages/patternfly-4/react-core/src/components/Dropdown/dropdownConstants.d.ts

```diff
@@ -16,7 +16,13 @@ export const DropdownContext: React.Context<{
   menuClass?: string,
   itemClass?: string,
   toggleClass?: string,
-  baseClass?: string
+  baseClass?: string,
+  baseComponent?: string,
+  sectionClass?: string,
+  sectionTitleClass?: string,
```

Comment:
Can you also add `sectionComponent` to this? It usually should be section but some components might use in future something else.